prepare_check_cleanup/reconcile.py

```diff
@@ -328,17 +426,27 @@ def check_delete(
     return True
 
 
-def cleanup(urls_with_actions: dict[str, str], discourse: Discourse) -> None:
+def cleanup(
+    urls_with_actions: dict[str, str], discourse: Discourse, github_token: str, repo: str
+) -> None:
     """Delete all URLs.
 
     Args:
         urls_with_actions: The URLs that had any actions against them.
         discourse: Client to the documentation server.
+        github_token: Token for communication with GitHub.
+        repo: The name of the repository.
     """
     for url in urls_with_actions.keys():
         with contextlib.suppress(DiscourseError):
             discourse.delete_topic(url=url)
 
+    with contextlib.suppress(GithubException):
```

Comment:
I think this can be changed a bit to have the cleanup log the failure and also report overall success. The suppression is a quick and easy way to ensure all the cleanup actions are taken. Should be implemented in the next push